src/components/Error-Fallback.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+function ErrorFallback() {
+  return (
+    <div className="bg-white p-8 rounded shadow-md">
```

Comment:
`bg-white p-8 rounded shadow-md`

Are these remnants of tailwind? Or is this how Chakra styles too?

Reply:
havent fully converted yet